modules/s2i/bash/artifacts/usr/local/s2i/assemble

```diff
@@ -14,3 +14,52 @@ source "${JBOSS_CONTAINER_JAVA_S2I_MODULE}/maven-s2i-overrides"
 # invoke the build
 maven_s2i_build
 
+#TODO: Clean this up to call the scripts instead
+
+shopt -s globstar
+echo "Setting environment variables"
+jarfile="/deployments/*.jar"
+libdir="/deployments/lib"
```

Comment:
hopefully we can use `$JAVA_LIB_DIR` here but need to double-check that it gets set prior to this code, in the event the user hasn't explicitly set it already.